content/Minecraft-Server-Hosting.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: article
+title: Install Minecraft Server
+description: 
+  So you want some of that blocky goodness? Well, get Java and some graphics cards and let's go!
+keywords:
+  - Minecraft
+  - Mojang
+  - Games
+hidden: false
+section: software
+---
+
+# Hosting A Minecraft Server
+
+Minecraft 1.17 dropped on June 8, 2021. In this article, we will be going over how to set up a Minecraft server on your Pop!_OS or Ubuntu install. A Minecraft System76 recommends having a minimum of 8Gb of RAM but for a good experience, 16Gb of RAM is recommended. Any hard drive over 128GB for a basic server but if you want to backups 1TB to 2TB is recommended. This article will live mostly in the terminal, if you want a refresher on the terminal you can see our article [here](https://support.system76.com/articles/terminal-basics).
+
+## Updating and Installing Software Needed
+
+To install the Minecraft server we will need to install Java (listed as default-jdk), wget, screen, and nmap.  Java is what Minecraft runs in and is required for the server to run - we are also marking it as hold so no future updates are applied (some java updates can break the server). wget will allow us to download the Minecrafter Server from a URL hosted by Mojang, this way we can put the file in the directory directly instead of using a web browser.  screen will allow us to run commands and the server in an environment we do not need to have open or be connected to - think of it as a virtual terminal instance you can connect and disconnect from.  nmap is a network scanner that we will use for testing and network configuration.
```

Comment:
Redundant wording: "Java is what Minecraft **runs** in and is required for the server to **run**"

Reply:
marking the java package as "held." or putting the java package on hold.

Reply:
comma splice: 

Server from a URL hosted by Mojang[.] Downloading Minecraft Server in this way means we can...

Reply:
. Screen will allow...

I think there's a clearer way of wording this. The virtual terminal explanation is a good explanation.

Reply:
OR `screen` and `nmap` to show them as Terminal name schemes.

Reply:
corrected spelling and grammar